regression/cbmc-java/virtual10/test.java

```diff
@@ -0,0 +1,33 @@
+interface A {
+  public int f();
```

Comment:
What is the importance of these methods? Seems like `Object` can only call `O::toString` or `C::toString`?